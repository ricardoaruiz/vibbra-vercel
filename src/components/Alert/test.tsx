import { renderWithThemeProvider } from 'utils/test'

import { Alert } from '.'

describe('<Alert />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(
      <Alert message="Altert message" />
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})
