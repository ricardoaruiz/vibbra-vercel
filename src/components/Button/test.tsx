import { renderWithThemeProvider } from 'utils/test'
import { Button } from '.'

describe('<Button />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<Button />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
