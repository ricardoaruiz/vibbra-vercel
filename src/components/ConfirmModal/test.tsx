import { renderWithThemeProvider } from 'utils/test'

import { ConfirmModal } from '.'

describe('<ConfirmModal />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<ConfirmModal />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
