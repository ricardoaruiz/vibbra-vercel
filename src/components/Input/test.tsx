import { renderWithThemeProvider } from 'utils/test'

import { Input } from '.'

describe('<Input />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<Input />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
