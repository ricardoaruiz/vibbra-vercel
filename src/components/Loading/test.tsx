import { renderWithThemeProvider } from 'utils/test'

import { Loading } from '.'

describe('<Loading />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<Loading />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
