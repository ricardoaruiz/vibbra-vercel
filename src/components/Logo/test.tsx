import { renderWithThemeProvider } from 'utils/test'

import { Logo } from '.'

describe('<Logo />', () => {
  it('should render correctly', () => {
    const { container } = renderWithThemeProvider(<Logo />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
