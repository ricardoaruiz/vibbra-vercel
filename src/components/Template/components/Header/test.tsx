import { renderWithThemeProvider } from 'utils/test'

import { Header } from '.'

describe('<Header />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<Header />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
