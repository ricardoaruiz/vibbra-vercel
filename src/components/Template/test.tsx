import { renderWithThemeProvider } from 'utils/test'

import { Template } from '.'

jest.mock('next/router', () => ({
  useRouter: () => ({
    push: jest.fn()
  })
}))

describe('<Template />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<Template />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
