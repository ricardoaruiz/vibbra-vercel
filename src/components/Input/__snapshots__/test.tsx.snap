// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> should render the heading 1`] = `
.c0 {
  height: 60px;
}

.c1 {
  border: 1px solid #8F8F8F;
  border-radius: 0.5rem;
  padding: 1rem 0.5rem;
  outline: none;
  font-size: 1.6rem;
  width: 100%;
  margin-bottom: 0.5rem;
}

.c1::-webkit-input-placeholder {
  color: #8F8F8F;
}

.c1::-moz-placeholder {
  color: #8F8F8F;
}

.c1:-ms-input-placeholder {
  color: #8F8F8F;
}

.c1::placeholder {
  color: #8F8F8F;
}

.c1:focus {
  border: 1px solid #26AD9E;
}

<main
  class="c0"
>
  <input
    class="c1"
    type="text"
  />
</main>
`
